Packages/com.llealloo.audiolink/Runtime/Scripts/ytdlpPlayer.cs

```diff
@@ -143,6 +141,25 @@ public static class ytdlpURLResolver
         private static string _ytdlpPath = "";
         private static bool _ytdlpFound = false;
 
+        private const string userDefinedYTDLPathKey = "YTDL-PATH-CUSTOM";
+        private const string userDefinedYTDLPathMenu = "Tools/AudioLink/Select Custom YTDL Location";
+
+        [MenuItem(userDefinedYTDLPathMenu, priority = 1)]
+        private static void SelectYtdlInstall()
+        {
+            string ytdlPath = EditorPrefs.GetString(userDefinedYTDLPathKey, "");
+            string tpath = ytdlPath.Substring(0, ytdlPath.LastIndexOf("/", StringComparison.Ordinal) + 1);
```

Comment:
Won't this break if you use `\` as the path separator?

Reply:
I did try using the `\` separator check but what I saw was that Unity was normalizing to `/` for the paths.